File: Meghana A S/2D Arrays/MatrixMultiplication.cpp
#include<bits/stdc++.h>
using namespace std;
int main() {
//     Write your code here.
int n1,m1;
cin>>n1>>m1;
int a1[n1][m1];
for(int i=0;i<n1;i++)
{
    for(int j=0;j<m1;j++)
    {
        cin>>a1[i][j];
    }
}
int n2,m2;
cin>>n2>>m2;
int a2[n2][m2];
for(int i=0;i<n2;i++)
{
    for(int j=0;j<m2;j++)
    {
        cin>>a2[i][j];
    }
}
int pro[m1][n2];
for(i=0;i<n1;i++)    
{    
for(j=0;j<m2;j++)    
{    
pro[i][j]=0;    
for(k=0;k<m2;k++)    
{    
pro[i][j]+=a[i][k]*b[k][j];    
}    
}    
}    
}

File: V.Vennela/Basics of Programming/Arrays2D.java/MatrixMultiplication.java
import java.io.*;
import java.util.*;

public class MatrixMultiplication{

public static void main(String[] args) throws Exception {
    // write your code here
    Scanner sc = new Scanner(System.in);
    int r1= sc.nextInt();
    int c1 = sc.nextInt();
    int a[][] = new int[r1][c1];
    for(int i=0;i<r1;i++)
    {
         for(int j=0;j<c1;j++)
        {
            a[i][j]=sc.nextInt();
        }
    }
    int r2 = sc.nextInt();
    int c2 = sc.nextInt();
    int b[][] = new int[r2][c2];
    for(int i=0;i<r2;i++)
    {
         for(int j=0;j<c2;j++)
        {
            b[i][j]=sc.nextInt();
        }
    }
    if(c1==r2)
    {
       int c[][] = new int[r1][c2];
       for(int i=0;i<r1;i++)
       {
           for(int j=0;j<c2;j++)
           {
               c[i][j]=0;
               for(int k=0;k<r2;k++)
               {
                   c[i][j]+=a[i][k]*b[k][j];
               }
               System.out.print(c[i][j]+" ");
           }
           System.out.println();
       }
    }
    else
    {
        System.out.println("Invalid input");
    }
    
 }

}
